benchmark/common.js

```diff
@@ -3,17 +3,18 @@
 const child_process = require('child_process');
 const http_benchmarkers = require('./_http-benchmarkers.js');
 
-exports.createBenchmark = function(fn, options) {
-  return new Benchmark(fn, options);
+exports.createBenchmark = function(fn, options, flags) {
```

Comment:
I think it would be better to make it `exports.createBenchmark = function(fn, config, options)`, where `options` is an object e.g. `{ flags: [] }`. I don't know why `config` is called `options` here, when it is not really optional.